src/rules/selector-no-redundant-nesting-selector/README.md

```diff
@@ -59,3 +59,51 @@ p {
   }
 }
 ```
+
+## Options
+
+Some false alarms can be blocked by assigning keywords:
+
+```js
+{
+  rules: {
+    'scss/selector-no-redundant-nesting-selector', [true, { keywords: ['when'] }],
```

Comment:
The standard way to name the option would be to call it `ignoreKeywords`. But that makes me think that is `keyword` the correct term for this? Is that what it's called in Less?

Reply:
I took it to mean reserved. I can follow your principle if you think it necessary

Reply:
Please have a look at this guide. It explains how to name options in stylelint rules:
https://stylelint.io/user-guide/rules/about#options